package.json

```diff
@@ -15,16 +15,19 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
-    "boom": "3.X.X",
-    "hoek": "4.X.X",
-    "joi": "9.X.X",
-    "wreck": "9.X.X"
+    "boom": "4.x.x",
+    "hoek": "4.x.x",
+    "joi": "9.x.x",
+    "wreck": "10.x.x"
+  },
+  "peerDependencies": {
+    "hapi": "14.x.x || 15.x.x"
   },
   "devDependencies": {
```

Comment:
could you explain more? I'm not very knowledgeable about the code-style here. What exactly should I do with the `.x.x` ?